tests/Tools/vmOpsTool/TestVirtualMachine.java

```diff
@@ -0,0 +1,8 @@
+class TestVirtualMachine {
+
+    String Name;
+    String NewSnapshot;
```

Comment:
Please see my updated pull request (second commit). If you'd like, I can merge the commits.